internal/a3m/a3m.go

```diff
@@ -190,6 +190,7 @@ func savePreservationTasks(
 			PreservationActionID: paID,
 		}
 		pt.StartedAt.Time = job.StartTime.AsTime()
```

Comment:
I think this would be easier to read with `StartedAt` assigned in the struct literal:

```go
pt := datatypes.PreservationTask{
	TaskID: job.Id,
	Name:   job.Name,
	Status: jobStatusToPreservationTaskStatus[job.Status],
	StartedAt: sql.NullTime{
		Time:  job.StartTime.AsTime(),
		Valid: true,
	},
	PreservationActionID: paID,
}
```